diff --git a/dist/index.cjs b/dist/index.cjs
index a0070d52224dcd965d008c99bca0702aa66270f7..9c5e309e398d0f254fd93611204a87f2fbbfe230 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -50,6 +50,9 @@ var __privateMethod = (obj, member, method) => {
   return method;
 };
 
+// Manually edited.
+var Buffer = require("buffer").Buffer;
+
 // src/index.ts
 var src_exports = {};
 __export(src_exports, {
diff --git a/dist/index.js b/dist/index.js
index c9f65fb9735280bf102e5ab82e3c48f41f0884d9..8c961c05e73236b5912636609c565ea3c4b68cee 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -30,6 +30,9 @@ var __privateMethod = (obj, member, method) => {
   return method;
 };
 
+// Manually edited.
+import { Buffer } from "buffer";
+
 // src/Account.ts
 function isAccount(account) {
   return typeof account === "object" && account != null && "getAddress" in account && account.getAddress instanceof Function && "getPublicKey" in account && account.getPublicKey instanceof Function && "sign" in account && account.sign instanceof Function;
diff --git a/package.json b/package.json
index 66a694afd84482a1f228d310c3aac4f7c29a5265..f2736c24fc64ec5c979ef379c143b7833c876b68 100644
--- a/package.json
+++ b/package.json
@@ -46,7 +46,8 @@
   },
   "dependencies": {
     "@noble/hashes": "^1.2.0",
-    "@noble/secp256k1": "^1.7.1"
+    "@noble/secp256k1": "^1.7.1",
+    "buffer": "^6.0.3"
   },
   "version": "4.4.2"
 }
